site/content/docs/cli/npm-packages.md

```diff
@@ -26,4 +29,58 @@ The latest runtime is `2022.10` at the time of writing. This runtime contains am
 - [Browse the latest runtime specs](https://www.checklyhq.com/docs/runtimes/specs/)
 - [Learn more about runtimes](https://www.checklyhq.com/docs/runtimes/)
 
+## Local Dependencies
+
+Browser Checks are also able to import other JavaScript and TypeScript files as dependencies.
+This can be particularly useful for defining helper functions to be reused across multiple checks.
```

Comment:
```suggestion
This is useful for defining helper functions to be reused across multiple checks.
```

Let's save some words. "Can" is a word that can often be dropped to save words. Shorter and bolder is easier to read. :)

Reply:
Thanks, good idea